storage/ProcessDatabaseSchema.ts

```diff
@@ -214,6 +214,13 @@ type ProcessDatabaseSchema = NamedSchema<
         mentalHealthOver65Notes: Note;
       };
     };
+
+    otherNotes: {
+      key: ProcessRef;
+      value: {
+        notes: Note;
+      };
```

Comment:
```suggestion
      value: Note;
```

No need to embed it any deeper I think.

Reply:
I initially tried this, but I think it will require a change to Remultiform, if I'm understanding correctly, as a `Note` type can't be assigned to a `StoreSchemaValue` (the only types accepted are here: https://github.com/LBHackney-IT/remultiform/blob/4c1f81c81fa30bc76ee6ccea41d93e018d56ad3d/src/database/types.ts#L26). 

Is it worth making the change, or leaving it embedded?

Reply:
If it failed to compile, probably not worth the effort. The other thing that might work is to change `Note` to be a `type` rather than an `interface`. But don't spend much time on it.

Reply:
You made it a `type` but didn't change this. Was that intentional?